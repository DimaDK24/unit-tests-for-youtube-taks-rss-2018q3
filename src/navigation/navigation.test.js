import Navigation from './navigation';

describe('Navigation', () => {
  it('.draw should wrap template with tag section.navitation', () => {
    // prepare - Arrange

    // action  - Act
    Navigation.draw();

    // check - Assert
    expect(document.querySelector('body>section.navigation')).not.toBe('');
  });
  it('render correct template into the section', () => {
    // prepare - Arrange

    // action  - Act
    Navigation.draw();

    // check - Assert
    expect(document.querySelector('body>section.navigation').innerHTML).toMatchSnapshot();
  });
});
