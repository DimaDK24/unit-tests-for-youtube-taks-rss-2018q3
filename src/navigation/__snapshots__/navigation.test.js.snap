// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation render correct template into the section 1`] = `
"
<div class=\\"navigation__number\\"></div>
<div class=\\"navigation__bar\\"></div>
"
`;
